src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/nonassociations/NonAssociationsReconciliationService.kt

```diff
@@ -115,10 +115,12 @@ class NonAssociationsReconciliationService(
         async { nonAssociationsApiService.getNonAssociationsBetween(id.offenderNo1, id.offenderNo2) }
     }.awaitBoth()
 
-    val nomisListSorted = nomisListUnsorted.sortedBy { it.effectiveDate }
+    val nomisListSortedBySequence = nomisListUnsorted.sortedBy { it.typeSequence }
```

Comment:
Needs to be same sort order as migration to eliminate the correct multiple open NAs